src/drivers/imx/sdma.c

```diff
@@ -443,6 +443,7 @@ static void sdma_disable_event(struct dma_chan_data *channel, int eventnum)
 static void sdma_channel_put(struct dma_chan_data *channel)
 {
 	struct sdma_chan *pdata = dma_chan_get_data(channel);
+
```

Comment:
This is not an unnecessary blank line. It's a blank line after declaration, it should stay in.